tools/coqdoc/coqdoc.css

```
@@ -327,3 +327,8 @@ ul.doclist {
     margin-top: 0em;
     margin-bottom: 0em;
 }
+
+:target {
```

```suggestion
.code :target {
```

To restrict the rule to code blocks. I'm not sure the same rule would work elsewhere.

---

Yes, much better.